src/storage/index/LookupBaseProcessor.inl

```diff
@@ -65,10 +65,10 @@ cpp2::ErrorCode LookupBaseProcessor<REQ, RESP>::requestCheck(const cpp2::LookupI
         yieldCols_ = *req.return_columns_ref();
     }
 
-    for (size_t i = 0; i < yieldCols_.size(); i++) {
-        resultDataSet_.colNames.emplace_back(yieldCols_[i]);
-        if (QueryUtils::toReturnColType(yieldCols_[i]) != QueryUtils::ReturnColType::kOther) {
-            deDupColPos_.emplace_back(i);
+    for (auto&& it : folly::enumerate(yieldCols_)) {
```

Comment:
It is perhaps more intuitive to use StructuredBinding. for example:
`auto&& [index, col] : folly::enumerate(yieldCols_)`